2996_reflection/reflection.md

```diff
@@ -3145,28 +3137,43 @@ Add a production to the grammar for `$nested-name-specifier$` as follows:
       ::
       $type-name$ ::
       $namespace-name$ ::
-      $computed-type-specifier$ ::
 +     $splice-namespace-qualifier$ ::
+      $computed-type-specifier$ ::
       $nested-name-specifier$ $identifier$ ::
       $nested-name-specifier$ template@~_opt_~@ $simple-template-id$ ::
 +
 + $splice-namespace-qualifier$:
-+     [: $constant-expression$ :]
++     $splice-specifier$
 ```
 :::
 
-Add a restriction to [expr.prim.id.qual]{.sref} for the constraint on `$constant-expression$`.
+Add a new paragraph restricting `$splice-namespace-qualifier$`, and renumber accordingly:
 
 ::: std
 ::: addu
-[0]{.pnum} For a `$splice-namespace-qualifier$`, the `$constant-expression$` shall be a converted constant expression of type `std::meta::info` that is a reflection of either a namespace or a namespace alias.
+[1]{.pnum} The `$splice-specifier$` of a `$splice-namespace-qualifier$` shall designate a namespace or namespace alias.
```

Comment:
Never renumber existing paragraphs. That's up to the editors. The existing numbers help reviewers look at the existing wording to line it up to check to make sure the change is right. 

That's why I tend to number paragraphs like `0`, `1 1/2`, `π`, whatever.

Reply:
That makes sense, thanks for the guidelines! I renumbered sections whose numbers I had changed.